dot/telemetry/afg_finalized_blocks_up_to.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package telemetry
+
+import (
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+//nolint
```

Comment:
why the nolint?

Reply:
must be because of the word `finalize`. Now that we have added setting to ignore linting warning for `finalize`, I can remove it.

Reply:
addressed